dashboards/azure-disk-storage/azure-disk-storage.json

```diff
@@ -71,8 +70,10 @@
             },
             "nrqlQueries": [
               {
-                "accountIds": [],
-                "query": "FROM Metric SELECT average(azure.compute.disks.CompositeDiskReadBytes.sec) AS 'Avg Composite Disk Read Bytes/sec' WHERE collector.name = 'azure-monitor' and azure.resourceType = 'microsoft.compute/disks' TIMESERIES AUTO "
+                "accountIds": [
+                  11817994
```

Comment:
As with the other file, we have a hard-coded `accountId` here. Unless intentional, we should remove this.